src/scenes/components/TimepointExplorer/TimepointListErrorButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { GrafanaTheme2 } from '@grafana/data';
+import { IconButton, useStyles2 } from '@grafana/ui';
+import { css } from '@emotion/css';
+
+import { useTimepointExplorerContext } from 'scenes/components/TimepointExplorer/TimepointExplorer.context';
+
+export function TimepointListErrorButton() {
```

Comment:
I created a new (scripted) check and got redirected to the dashboard, where I saw the "there was an error fetching data" button as there was no data yet. After some time I did get a timepoint in the Timepoint List, but the refresh button didn't go away until I refreshed the page which was a bit confusing
<img width="1264" height="723" alt="image" src="https://github.com/user-attachments/assets/e3ce60b3-d46e-496b-95e8-d16f18548072" />